backend/clubs/management/commands/send_emails.py

```diff
@@ -198,9 +208,9 @@ def handle(self, *args, **kwargs):
                                     invite.send_owner_invite()
                                 else:
                                     invite.send_mail()
-                        elif action == "fair":
+                        elif action == "physical_fair":
                             send_fair_email(club, receiver)
-                        elif action == "postfair":
+                        elif action == "physical_postfair":
```

Comment:
I'm not sure why we're seeing this in the diff, perhaps you need to rebase on master again?